tango.py

```diff
@@ -207,6 +207,25 @@ def getInfo(self):
 
         return stats
 
+    def getPartialOutput(self, jobid):
+        """getPartialOutput - Return the partial output of a job"""
+        try:
+            jobInfo = self.jobQueue.liveJobs.get(jobid)
+
+            if jobInfo is None:
+                raise Exception("Invalid job id")
+
+            vm = jobInfo.vm
+
+            if not jobInfo.assigned or vm is None:
+                raise Exception("Job %s is not running yet" % jobid)
```

Comment:
should we return a different status id for jobs that are on the queue but not running yet, so that we can display this info to the user on the front end?